Samples.xcodeproj/project.pbxproj

```diff
@@ -1484,7 +1484,6 @@
 				00181B462846AD7100654571 /* View+Alert.swift in Sources */,
 				E0FE32E728747778002C6ACA /* BrowseBuildingFloorsView.swift in Sources */,
 				D752D95F2A3BCE06003EB25E /* DisplayMapFromPortalItemView.swift in Sources */,
-				1CAB8D072A3CCBD8002AA649 /* SetVisibilityOfSubtypeFeatureLayerView.Views.swift in Sources */,
```

Comment:
This change shouldn't be here. Did you accidentally removed it from "Compile Sources" build phase?

Reply:
No it is still in there. I couldn't figure out why or how that it was removed. It seemed to happen when I locally added the toolkit daily build. I tried start over from scratch, and it still did it.

Reply:
OK, it seems to be related to https://github.com/Esri/arcgis-maps-sdk-swift-samples/pull/205#discussion_r1256448426 . Typically it is good to go in a separate PR but I'm OK to get it fixed here.